cabal.project

```diff
@@ -0,0 +1 @@
+packages: .
```

Comment:
What do you think about the `cabal.project.dist` pattern? I kind of like that for libraries (compared to top-level executables) because if you are in `crucible/deps/golang` making updates, your build commands will be able to build `crucible-go` without you having to change directories (because `cabal` won't pick up the spurious `cabal.project` file

Reply:
Good point. I've renamed this to `cabal.project.dist`, and I've added `cabal.project` to `.gitignore` to prevent `ln -s cabal.project.dist cabal.project` from dirtying the `git` tree.